crypto/evp/evp_fetch.c

```diff
@@ -23,24 +23,6 @@
 
 #define NAME_SEPARATOR ':'
 
-static void evp_method_store_free(void *vstore)
-{
-    ossl_method_store_free(vstore);
-}
-
-static void *evp_method_store_new(OSSL_LIB_CTX *ctx)
-{
-    return ossl_method_store_new(ctx);
```

Comment:
Sorry, didn't spot this inconsistency before. It is strange that we completely remove `evp_method_store_new` and `evp_method_store_free` in order to call `ossl_method_store_new` directly in context.c. However, we don't do the same for `ossl_encoder_store_new`, `ossl_decoder_store_new` or `ossl_loader_store_new`.